tests/kernel/smp/src/main.c

```diff
@@ -14,11 +14,12 @@
 #error SMP test requires at least two CPUs!
 #endif
 
+#define MAX_NUM_THREADS CONFIG_MP_MAX_NUM_CPUS
```

Comment:
A fix for ? You do not describe the actual issue the pre-defined value was generating. Describe it in your commit message